features/polyExplorer/src/context/explorer-context.jsx

```diff
@@ -83,6 +83,8 @@ const loadStoriesMetadata = () => {
     };
 };
 
+const intermediateScreens = ["/search", "/entity-details"];
```

Comment:
Also, on second look: Is `/entity-details` actually the _filter screen_? Isn't it `/entity-filters`? (But if it works, it works.)

Reply:
You are right, yet it doesn't matter at all here since filters always go back to main where no back-option is given anyway. So I'll get rid of it entirely here